fieldservice_sale/models/sale_order_line.py

```diff
@@ -31,6 +33,26 @@ def _compute_product_updatable(self):
             else:
                 super(SaleOrderLine, line)._compute_product_updatable()
 
+    @api.multi
+    @api.depends('product_id')
+    def _compute_qty_delivered_method(self):
+        super(SaleOrderLine, self)._compute_qty_delivered_method()
+        for line in self:
+            if not line.is_expense and line.product_id.type == 'service' \
+               and line.product_id.field_service_tracking == 'order':
+                line.qty_delivered_method = 'field'
+
+    @api.multi
+    @api.depends('fsm_order_id.stage_id')
+    def _compute_qty_delivered(self):
```

Comment:
May be dumb question, but are you sure it will not interfere with https://github.com/odoo/odoo/blob/12.0/addons/sale_stock/models/sale_order.py#L202 ?

Reply:
I am not 100% positive.  But [https://github.com/odoo/odoo/blob/12.0/addons/sale_timesheet/models/sale_order.py#L142](https://github.com/odoo/odoo/blob/12.0/addons/sale_timesheet/models/sale_order.py#L142) does something very similar to what I did, so I assume there will be no issues here.